content/shmem_wait_nbe.tex

```diff
@@ -0,0 +1,36 @@
+\apisummary{
+    The request wait routine provides a method for completing
+    outstanding operations on a request.
```

Comment:
Does "completing" (and "completed", below) imply local completion (i.e., the local buffer can be [re]used) or remote completion?